graviti/dataframe/frame.py

```diff
@@ -806,6 +835,85 @@ def _pylist_to_pyarrow(values: Iterable[Any], schema: pt.PortexRecordBase) -> pa
 
         return pa.array(processor(values), schema.to_pyarrow())
 
+    @staticmethod
+    def _pylist_to_pyarrow_and_pydict(
+        values: Iterable[Any], schema: pt.PortexRecordBase
+    ) -> Tuple[pa.StructArray, Dict[str, Any]]:
+        pyarrow_schema = _to_simple_pyarrow(schema)
+        pyarrow_array = pa.array(values, pyarrow_schema)
+
+        pydict = DataFrame._get_pydict(values, schema)
+
+        return pyarrow_array, pydict
+
+    @staticmethod
+    def _get_pydict(values: Iterable[Any], schema: pt.PortexRecordBase) -> Dict[str, Any]:
+        pydict: Dict[str, Any] = {}
+        for key, portex_type in schema.items():
+            container = portex_type.container
+            if issubclass(container, ArraySeries) and isinstance(
```

Comment:
what if the items of array is enum or array or file?

Reply:
After testing, there was no problem

Reply:
I just test the case that enum inside array, and it failed:
```python
In [13]: schema = pt.record({"a": pt.array(pt.enum(["cat", "dog"]))})

In [14]: data = [{"a": ["cat", "dog"]}] * 3

In [15]: DataFrame(data, schema)
---------------------------------------------------------------------------
ArrowInvalid                              Traceback (most recent call last)
Input In [15], in <cell line: 1>()
----> 1 DataFrame(data, schema)

File ~/workspace/phabricator/graviti-python-sdk/graviti/dataframe/frame.py:146, in DataFrame.__new__(cls, data, schema)
    142     return cls.from_pyarrow(pa.array(data))
    144 if isinstance(data, list):
    145     return cls._from_pyarrow_and_pydict(
--> 146         *cls._pylist_to_pyarrow_and_pydict(data, schema), schema
    147     )
    149 raise ValueError("DataFrame only supports creating from list object now")

File ~/workspace/phabricator/graviti-python-sdk/graviti/dataframe/frame.py:843, in DataFrame._pylist_to_pyarrow_and_pydict(values, schema)
    838 @staticmethod
    839 def _pylist_to_pyarrow_and_pydict(
    840     values: Iterable[Any], schema: pt.PortexRecordBase
    841 ) -> Tuple[pa.StructArray, Dict[str, Any]]:
    842     pyarrow_schema = _to_simple_pyarrow(schema)
--> 843     pyarrow_array = pa.array(values, pyarrow_schema)
    845     pydict = DataFrame._get_pydict(values, schema)
    847     return pyarrow_array, pydict

File ~/.pyenv/versions/3.8.13/lib/python3.8/site-packages/pyarrow/array.pxi:316, in pyarrow.lib.array()

File ~/.pyenv/versions/3.8.13/lib/python3.8/site-packages/pyarrow/array.pxi:39, in pyarrow.lib._sequence_to_array()

File ~/.pyenv/versions/3.8.13/lib/python3.8/site-packages/pyarrow/error.pxi:144, in pyarrow.lib.pyarrow_internal_check_status()

File ~/.pyenv/versions/3.8.13/lib/python3.8/site-packages/pyarrow/error.pxi:100, in pyarrow.lib.check_status()

ArrowInvalid: Could not convert 'cat' with type str: tried to convert to int8
```